apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -195,7 +195,7 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
   const handleSellTransaction = (operation: Operation, result: ResultsState) => {
     handleTransactionResult(result, operation);
   };
-  const isDemo3Enabled = useIsSwapLiveFlagEnabled("ptxSwapLiveAppDemoThree");
+  const isDemo3Enabled = true;
```

Comment:
IsSwapLiveAppEnabled would be a better name here

Reply:
If this is a hardcoded const do we need it?

Reply:
removed